test/e2e/ha/scale_core_contrail_services_test.go

```diff
@@ -317,6 +332,159 @@ func TestHACoreContrailServices(t *testing.T) {
+			schematransformerContainer.Image = "registry:5000/contrail-nightly/contrail-controller-config-schema:" + targetVersionMap["cemVersion"]
+			servicemonitorContainer.Image = "registry:5000/contrail-nightly/contrail-controller-config-svcmonitor:" + targetVersionMap["cemVersion"]
+			analyticsapiContainer.Image = "registry:5000/contrail-nightly/contrail-analytics-api:" + targetVersionMap["cemVersion"]
+			collectorContainer.Image = "registry:5000/contrail-nightly/contrail-analytics-collector:" + targetVersionMap["cemVersion"]
+			queryengineContainer.Image = "registry:5000/contrail-nightly/contrail-analytics-query-engine:" + targetVersionMap["cemVersion"]
+			//statusmonitorContainer.Image = "registry:5000/contrail-operator/engprod-269421/contrail-statusmonitor:" + intendedVersionMap["contrail-statusmonitor"]
+
+			webuijobContainer := utils.GetContainerFromList("webuijob", instance.Spec.Services.Webui.Spec.ServiceConfiguration.Containers)
+			webuiwebContainer := utils.GetContainerFromList("webuiweb", instance.Spec.Services.Webui.Spec.ServiceConfiguration.Containers)
+			webuijobContainer.Image = "registry:5000/contrail-nightly/contrail-controller-webui-job:" + targetVersionMap["cemVersion"]
+			webuiwebContainer.Image = "registry:5000/contrail-nightly/contrail-controller-webui-web:" + targetVersionMap["cemVersion"]
+
+			pmContainer := utils.GetContainerFromList("provisioner", instance.Spec.Services.ProvisionManager.Spec.ServiceConfiguration.Containers)
+			pmContainer.Image = "registry:5000/contrail-operator/engprod-269421/contrail-operator-provisioner:" + targetVersionMap["contrail-operator-provisioner"]
+
+			err = f.Client.Update(context.TODO(), instance)
+			require.NoError(t, err)
+
+			t.Run("then Zookeeper has updated image", func(t *testing.T) {
+				err := wait.Contrail{
+					Namespace:     namespace,
+					Timeout:       5 * time.Minute,
+					RetryInterval: retryInterval,
+					Client:        f.Client,
+					Logger:        log,
+				}.ForPodImageChange(f.KubeClient, "zookeeper", zkContainer.Image, "zookeeper")
+				require.NoError(t, err)
+			})
+
+			t.Run("then Zookeeper StatefulSet should be ready", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-zookeeper-zookeeper-statefulset", replicas))
+			})
+
+			t.Run("then Rabbitmq has updated image", func(t *testing.T) {
+				err := wait.Contrail{
+					Namespace:     namespace,
+					Timeout:       5 * time.Minute,
+					RetryInterval: retryInterval,
+					Client:        f.Client,
+					Logger:        log,
+				}.ForPodImageChange(f.KubeClient, "rabbitmq", rmqContainer.Image, "rabbitmq")
+				require.NoError(t, err)
+			})
+
+			t.Run("then Rabbitmq StatefulSet should be ready", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-rabbitmq-rabbitmq-statefulset", replicas))
+			})
+
+			t.Run("then Cassandra has updated image", func(t *testing.T) {
+				t.Skip()
+				/*err := wait.Contrail{
```

Comment:
If test is skipped, then it doesn't have to be commented out

Reply:
Done